Core/Src/bms.c

```diff
@@ -59,9 +59,12 @@ void vBMSCANMonitor(void *pv_params)
 			/*TO-DO: fix duration (ticks)*/
 			osTimerStart(bms->bms_monitor_timer,
 				     BMS_CAN_MONITOR_DELAY);
-			bms->dcl = (uint16_t)((msg_from_queue.data[1] << 8) &
-					      msg_from_queue.data[0]);
 			//serial_print("BMS DCL %d", bms->dcl);
+			if (msg_from_queue.id == BMS_CURR_LIMIT_MSG) {
```

Comment:
do we need the BMS ccl like ever? if not lets yeat it and just keep the watchdog

Reply:
yeah I was gonna use CCL but the AC/DC thing popped up again